tasks/cluster_agent.py

```diff
@@ -123,3 +127,38 @@ def image_build(ctx):
     shutil.copy2(latest_file, "Dockerfiles/cluster-agent/")
     ctx.run("docker build -t {} Dockerfiles/cluster-agent".format(AGENT_TAG))
     ctx.run("rm Dockerfiles/cluster-agent/datadog-cluster-agent")
+
+
+@task
+def add_prelude(ctx, new_version):
```

Comment:
Both `add_prelude` and `update_changelog` only work on linux. I don't know if it's a limitation we want to change, but we should add a comment in the docstring.